MediaBrowser.Model/SyncPlay/PlayQueueUpdateReason.cs

```diff
@@ -0,0 +1,58 @@
+namespace MediaBrowser.Model.SyncPlay
+{
+    /// <summary>
+    /// Enum PlayQueueUpdateReason.
+    /// </summary>
+    public enum PlayQueueUpdateReason
+    {
+        /// <summary>
+        /// A user is requesting to play a new playlist.
+        /// </summary>
+        NewPlaylist = 0,
+
+        /// <summary>
+        /// A user is changing the playing item.
+        /// </summary>
+        SetCurrentItem = 1,
+
+        /// <summary>
+        /// A user is removing items from the playlist.
+        /// </summary>
+        RemoveItems = 2,
+
+        /// <summary>
+        /// A user is moving an item in the playlist.
+        /// </summary>
+        MoveItem = 3,
+
+        /// <summary>
+        /// A user is making changes to the queue.
+        /// </summary>
+        Queue = 4,
+
+        /// <summary>
+        /// A user is making changes to the queue.
```

Comment:
What is the difference between `Queue` and `QueueNext`?

Reply:
With `QueueNext` items are added after the playing item instead of the end of the playlist. The docs should be more helpful now.